src/screens/AppIntroScreen.tsx

```diff
@@ -86,7 +160,7 @@ const TermsAndConditionsSection = ({ setShowButtonTermsAndConditions }) => {
             params: {
               title: texts.profile.privacyCheckLink,
               query: QUERY_TYPES.PUBLIC_HTML_FILE,
-              queryVariables: { name: 'datenschutzProfile' }
+              queryVariables: { name: contentName }
```

Comment:
where does the `contentName` comes from? is this something we need to create for existing apps?

Reply:
If we have the `termsAndConditions` option in the slider, then the name of the content we want to show must be added to the item of the slider with the `contentName` key. Otherwise the `datenschutzProfile` content always needs to be created.

Reply:
seems legit, all right